data/netcdf_data.py

```diff
@@ -718,22 +730,39 @@ def variables(self) -> VariableList:
         if url.endswith(".sqlite3"):
             with SQLiteDatabase(url) as db:
                 self._variable_list = db.get_data_variables()  # Cache the list for later
+        
+        elif url.endswith(".zarr"):
+            ds_zarr = xarray.open_zarr(url)
+            var_list =[]
+            for var in list(ds_zarr.data_vars):
+
+                name = var
+                units = ds_zarr.variables[var].attrs['units'] if ds_zarr.variables[var].attrs['units'] else None
+                long_name = ds_zarr.variables[var].attrs['long_name'] if ds_zarr.variables[var].attrs['long_name'] else name
+                valid_min = ds_zarr.variables[var].attrs['valid_min'] if ds_zarr.variables[var].attrs['valid_min'] != 1.17549e-38 else None
+                valid_max = ds_zarr.variables[var].attrs['valid_max'] if ds_zarr.variables[var].attrs['valid_max'] != 3.40282e+38 else None
+
+                var_list.append(Variable(name, long_name, units, list(itertools.chain(*[ds_zarr[name].dims])), valid_min, valid_max))
```

Comment:
is `list(iterools.chain(*[ds_zarr[name].dims))` really needed? Isn't `list(ds_zarr[name.dims])` sufficient?

Reply:
Yes, it is.
Modified the code accordingly in the new commit.